### PR TITLE
Add OpenAIP map provider

### LINKED ISSUE #372: OpenAIP
Hello,

Adding [OpenAIP tiles](https://www.openaip.net/) sould probably be considered.

See maps https://maps.openaip.net/

See usage https://github.com/hankhank10/MSFS2020-cockpit-companion/blob/5ee36a84408aded2fa8e5e9c5b13df95a11f1a39/templates/glass.html#L512-L520

Kind regards

### PR DESCRIPTION
**Pull request by scls19fr**
Closes #372

### CODE REVIEW

**Comment by brunob** [comment]
@jieter any thoughts ? (i'll put this one as overlay in preview code after merge)

**Comment by jieter** [comment]
Looks good!

File: leaflet-providers.js [hunk 1]
```diff
@@ -1031,6 +1031,20 @@
...
+			url: 'http://{s}.tile.maps.openaip.net/geowebcache/service/tms/1.0.0/openaip_basemap@EPSG%3A900913@png/{z}/{x}/{y}.{ext}',
+			options: {
+				attribution: '<a href="https://www.openaip.net/">openAIP Data</a> (<a href="https://creativecommons.org/licenses/by-sa/3.0/">CC-BY-NC-SA</a>)',
+				ext: 'png',
+				minZoom: 4,
+				maxZoom: 14,
+				tms: true,
+				detectRetina: true,
+				subdomains: '12',
+				format: 'image/png',
```
Comment (jieter) [inline leaflet-providers.js:1045]:
```suggestion
```

File: leaflet-providers.js [hunk 2]
```diff
@@ -1031,6 +1031,19 @@
...
+			url: 'http://{s}.tile.maps.openaip.net/geowebcache/service/tms/1.0.0/openaip_basemap@EPSG%3A900913@png/{z}/{x}/{y}.{ext}',
+			options: {
+				attribution: '<a href="https://www.openaip.net/">openAIP Data</a> (<a href="https://creativecommons.org/licenses/by-sa/3.0/">CC-BY-NC-SA</a>)',
+				ext: 'png',
+				minZoom: 4,
+				maxZoom: 14,
+				tms: true,
+				detectRetina: true,
+				subdomains: '12',
+				transparent: true
```
Comment (jieter) [inline leaflet-providers.js:1045]:
```suggestion
				subdomains: '12'
```

**Pull request merged**